server/backend/backend.go

```diff
@@ -28,7 +28,7 @@ import (
 	jsoncdc "github.com/onflow/cadence/encoding/json"
 	sdk "github.com/onflow/flow-go-sdk"
 	"github.com/onflow/flow-go/access"
-	fvmerrors "github.com/onflow/flow-go/fvm/errors"
+	fvmerrors "github.com/onflow/flow-go/fvm/errors" // nolint:typecheck
```

Comment:
why do we need this ignored?

Reply:
the linter is incorrectly reporting this as an error (i didn't want to spent too much time looking into this)

Reply:
if the linter has these bugs shouldn't we rather just use an older version then?

Reply:
looks like the linter was compiled using the wrong go version.  these are fixed now